### DISCUSSION TITLE
Load image and text in safari very slow.

### DISCUSSION THREAD

**Discussion by nguyen-quang-phu** [vercel/next.js#33230] [Help]
Hello everyone, I just learned nextjs recently, when i run web in chrome, everythings seems good. But i run web in safari, it very low for load image and text, i will try use sharp for production https://nextjs.org/docs/messages/sharp-missing-in-production. and i adjusted lazy_bound for image but it not work.Does anyone have a solution for this? This is my website: https://synplan.ai/

https://user-images.githubusercontent.com/43996827/149203778-208f6fa3-573c-45c9-9fa3-50f8e765b810.mov

https://user-images.githubusercontent.com/43996827/149203809-5d85307d-5d9b-4137-b31a-a5d2574b660a.mov

.

**Comment by wwwdepot** [comment]
Image component doesn't process vector/animated images. Just use regular image files (jpeg, png, webp or avif).

https://github.com/vercel/next.js/blob/628e6aca97c845305cfcd8055751d9e88e686615/packages/next/server/image-optimizer.ts#L306-L329
